internal/provider/cluster.go

```diff
@@ -308,25 +308,25 @@ func resourceClusterRead(ctx context.Context, d *schema.ResourceData, meta inter
 }
 
 func setClusterKVs(d *schema.ResourceData, cluster *model.Cluster, p *scylla.CloudProvider) error {
-	d.Set("cluster_id", cluster.ID)
-	d.Set("name", cluster.ClusterName)
-	d.Set("cloud", p.CloudProvider.Name)
-	d.Set("region", cluster.Region.ExternalID)
-	d.Set("node_count", len(model.NodesByStatus(cluster.Nodes, "ACTIVE")))
-	d.Set("user_api_interface", cluster.UserAPIInterface)
-	d.Set("node_type", p.InstanceByID(cluster.Datacenter.InstanceID).ExternalID)
-	d.Set("node_dns_names", model.NodesDNSNames(cluster.Nodes))
-	d.Set("node_private_ips", model.NodesPrivateIPs(cluster.Nodes))
-	d.Set("cidr_block", cluster.Datacenter.CIDRBlock)
-	d.Set("scylla_version", cluster.ScyllaVersion.Version)
-	d.Set("enable_vpc_peering", !strings.EqualFold(cluster.BroadcastType, "PUBLIC"))
-	d.Set("enable_dns", cluster.DNS)
-	d.Set("datacenter_id", cluster.Datacenter.ID)
-	d.Set("datacenter", cluster.Datacenter.Name)
-	d.Set("status", cluster.Status)
+	_ = d.Set("cluster_id", cluster.ID)
+	_ = d.Set("name", cluster.ClusterName)
+	_ = d.Set("cloud", p.CloudProvider.Name)
+	_ = d.Set("region", cluster.Region.ExternalID)
+	_ = d.Set("node_count", len(model.NodesByStatus(cluster.Nodes, "ACTIVE")))
+	_ = d.Set("user_api_interface", cluster.UserAPIInterface)
+	_ = d.Set("node_type", p.InstanceByID(cluster.Datacenter.InstanceID).ExternalID)
+	_ = d.Set("node_dns_names", model.NodesDNSNames(cluster.Nodes))
+	_ = d.Set("node_private_ips", model.NodesPrivateIPs(cluster.Nodes))
+	_ = d.Set("cidr_block", cluster.Datacenter.CIDRBlock)
+	_ = d.Set("scylla_version", cluster.ScyllaVersion.Version)
+	_ = d.Set("enable_vpc_peering", !strings.EqualFold(cluster.BroadcastType, "PUBLIC"))
+	_ = d.Set("enable_dns", cluster.DNS)
+	_ = d.Set("datacenter_id", cluster.Datacenter.ID)
+	_ = d.Set("datacenter", cluster.Datacenter.Name)
+	_ = d.Set("status", cluster.Status)
```

Comment:
Bump go to 1.20 and errors.Join it?

Reply:
But I would address it in next PR
```
var diags diag.Diagnostics
err = d.Set("status", cluster.Status)
if err != nil {sdkdiag.AppendErrorf(diags, "creating API Gateway VPC Link (%s): %s", d.Get("name").(string), err)}
```

Reply:
removed this linter and related changes

Reply:
Just my $0.02, IMO we should keep `errcheck` and simply discard the error from `Set`(to satisfy the linter) if we don't care about handling it, I can imagine other error checks that are more releveant than `Set` ones and should be handled

Reply:
Agree, but we should do that in seperate PR

Reply:
> Agree, but we should do that in seperate PR

Sure, we can, since we don't have any CI on main so disabling part of the linter would not worsen the quality of checks.

Please create an issue and put it in a todo comment next to a part of code you disable the errcheck. Alternatively add a todo that errcheck should be also enabled, but it's not due to X issue.

Reply:
Done